megabeast/plotting/plot_graphic_model.py

```diff
@@ -0,0 +1,93 @@
+from graphviz import Digraph
+import numpy as np
+
+__all__ = ["create_graphic_model", "plot_graphic_model"]
+
+
+def create_graphic_model(nodes, edges, type):
```

Comment:
Codacy found an issue: [Redefining built-in 'type'](https://app.codacy.com/manual/kgordon/megabeast/pullRequest?prid=6302069)